MericCelik/assignment3/Trie.java

```diff
@@ -0,0 +1,50 @@
+
+/**@author AycaMericCelik
+ * */
+public class Trie {
+    private TrieNode root;
+
+    public Trie(){
+        this.root = new TrieNode();
+    }
+
+    public Trie(String[] words){
+        this.root = new TrieNode();
+        for(String word:words)
+            this.addWord(word);
+    }
+
+    public void addWord(String word)
+    {
```

Comment:
Why the C formatting?

Reply:
I don't know very much about it, is it the formatting above?
public foo()
{
//code
}

Actually I have always written code like this since couple of months ago. :smile: Why is it called C formatting and is it better to avoid it?